drivers/gles3/shaders/tonemap_inc.glsl

```diff
@@ -76,18 +84,86 @@ vec3 tonemap_aces(vec3 color, float p_white) {
 	return color_tonemapped / p_white_tonemapped;
 }
 
-// Based on Reinhard's extended formula, see equation 4 in https://doi.org/cjbgrt
-vec3 tonemap_reinhard(vec3 color, float p_white) {
-	float white_squared = p_white * p_white;
-	vec3 white_squared_color = white_squared * color;
-	// Equivalent to color * (1 + color / white_squared) / (1 + color)
-	return (white_squared_color + color * color) / (white_squared_color + white_squared);
+// Mean error^2: 3.6705141e-06
+vec3 agx_default_contrast_approx(vec3 x) {
+	vec3 x2 = x * x;
+	vec3 x4 = x2 * x2;
+
+	return +15.5 * x4 * x2 - 40.14 * x4 * x + 31.96 * x4 - 6.868 * x2 * x + 0.4298 * x2 + 0.1191 * x - 0.00232;
+}
+
+vec3 agx(vec3 val) {
+	const mat3 agx_mat = transpose(mat3(
+			0.544813, 0.37379614, 0.08139087,
+			0.14041554, 0.75414325, 0.10544122,
+			0.0888119, 0.17888511, 0.73230299));
+
+	// AgX does not provide whitepoint adjustments, so hardcode it to a value that matches the Blender appearance closely.
+	const float white = 16.016004;
+
+	const float min_ev = -12.47393;
+	const float max_ev = log2(white);
+
+	// Input transform (inset).
+	val = agx_mat * val;
+
+	// Log2 space encoding.
+	val = clamp(log2(val), min_ev, max_ev);
+	val = (val - min_ev) / (max_ev - min_ev);
+
+	// Apply sigmoid function approximation.
+	val = agx_default_contrast_approx(val);
+
+	return val;
+}
+
+vec3 agx_eotf(vec3 val) {
+	const mat3 agx_mat_inv = transpose(mat3(
+			1.96489403, -0.85600791, -0.10888612,
+			-0.29930908, 1.32639189, -0.02708281,
+			-0.16435644, -0.2382074, 1.40256385));
+
+	// Inverse input transform (outset).
+	val = agx_mat_inv * val;
+
+	// sRGB IEC 61966-2-1 2.2 Exponent Reference EOTF Display
+	// NOTE: We're linearizing the output here. Comment/adjust when
+	// *not* using a sRGB render target.
+	val = pow(val, vec3(2.2));
```

Comment:
This might be a big part of why this is so different from the other results. We are linearizing the color with an improper formula, then immediately converting from linear -> gamma using the proper formula.